platform_versioned_docs/version-24.2/enterprise/advanced-topics/azure-batch-standup.mdx

```diff
@@ -0,0 +1,461 @@
+
+Disadvantages:
+
+- Spot and low-priority nodes can be preempted, which can cause the pipeline to fail.
+
+### 6. Use Entra authentication
+
+Docs: https://www.nextflow.io/docs/latest/azure.html#microsoft-entra
+
+#### Create service principal for the platform to authenticate as
+1. Create service principal in Azure (https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal)
+1. Assign roles to the service principal (https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current)
+1. Get Service Principal ID, Tenant ID and Client Secret (https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal#option-3-create-a-new-client-secret)
+1. Add to Seqera Platform credentials (https://docs.seqera.io/platform/24.2/compute-envs/azure-batch#entra-service-principal)
+
+On Seqera Platform:
+
+1. Add new credentials with the name `entra-keys` and select the type "Azure"
+1. Add the Service Principal ID, Tenant ID and Client Secret
+1. Click "Create" to save the credentials
+
+#### Create a managed identity for Nextflow to authenticate as
+
+Back on the Azure Portal:
+
+1. Create a managed identity https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp
+1. Assign the relevant roles to the managed identity (https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current and https://www.nextflow.io/docs/latest/azure.html#required-role-assignments)
+1. Note the managed identity client ID for later
+1. Go to the Azure Portal, go to the Batch account you created earlier.
+1. Go to the "Pools" tab and find the pool called `tower-pool-${id}`, where `${id}` is the ID of the head node pool you made earlier
+1. Click "Identity"
+1. Click "Add User Assigned Identity"
+1. Select the managed identity you created earlier
+1. Click "Add"
+
+Processes running on this pool will can now use the managed identity to authenticate to Azure Batch and Storage.
+
+On Seqera Platform:
+
+1. Add a new compute environment with the name `entra-mi` and select the type `Azure Batch`
+1. Select the location as the same region as your Batch account
+1. Select the "Config mode" as "Manual"
+1. Select the Compute pool name as the pool you added the managed identity to earlier, `tower-pool-${id}`
+1. For "Managed Identity Client ID", put the client ID of the managed identity you created earlier
+
+Duplicate the `hello-world-autopool-low-priority` pipeline and save it as `hello-world-entra-mi`.
+
+Launch the pipeline by clicking "Launch".
+
+The pipeline will run as before, but using the managed identity to authenticate to Azure Batch and Storage. No keys or storage required.
+
+> [!NOTE]
+> If we wishes, we could use User Subscription mode instead of Batch Managed here although this is out of the scope of this tutorial.
```

Comment:
```suggestion
> If we wished, we could use User Subscription mode instead of Batch Managed here although this is out of the scope of this tutorial.
```